packages/util/src/assignment.ts

```diff
@@ -11,7 +11,7 @@ import { parseDegree } from "@encointer/types";
 import {u64, Vec} from "@polkadot/types";
 import {Option} from "@polkadot/types-codec";
 import type {Moment} from "@polkadot/types/interfaces/runtime";
-import assert from "assert";
+// import assert from "assert";
```

Comment:
This is a node-js type and it gave me some polyfill errors in the integritee-campaign page